STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -480,6 +480,18 @@ certificate has been issued by the CA, the IdO:
 ~~~
 {: #fig-star-ido-order-resource-updated title="STAR Order Resource Updated on IdO"}
 
+Before forwarding the Order request to the CA, the IdO SHOULD ensure that the
```

Comment:
Maybe start with a high-level motivation, something like: 
> This delegation protocol is predicated on the NDC being able to fetch certificates periodically using an unauthenticated HTTP GET, since in general the NDC does not possess an account on the CA and therefore cannot issue the standard POST-as-GET ACME request.

Reply:
I thought about this and found that we have something to that purpose already in the last para of the Overview section.  Quoting:
```
Also note that the successful negotiation of the "unauthenticated GET" (Section
3.4 of {{!RFC8793}}) is required in order to allow the NDC to access the
`star-certificate` URL on the CA.
```
Is that too far in the reader's stack and therefore in need to be refreshed?

Reply:
Yes, I think it needs to be refreshed. I'm not sure it was clear to our reviewer.

Reply:
done in 11a8c38